lib/slider/index.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component, PropTypes } from 'react';
 import Slider from 'react-slick';
-import { SliderArrowRight, SliderArrowLeft } from './arrows.js';
+// import { SliderArrowRight, SliderArrowLeft } from './arrows.js';
```

Comment:
@nikhilaravi No Idea, they were working, maybe something was lost in a merge. I will take a look :+1: